### ISSUE TITLE
Functions not working

### ISSUE THREAD

**Issue by 4artiseth** [adrianhajdin/pricewise#2]
![image](https://github.com/adrianhajdin/pricewise/assets/127159714/916cd6f7-a090-4e02-ac86-b578c323e88b)
![image](https://github.com/adrianhajdin/pricewise/assets/127159714/d7c534e1-7939-4b10-a2b5-4caa7a803109)
Buttons not functioning.

**Comment by Greg-Mukuria** [comment]
true

**Comment by NamanMalhotra1706** [comment]
> ![image](https://user-images.githubusercontent.com/127159714/271940964-916cd6f7-a090-4e02-ac86-b578c323e88b.png) ![image](https://user-images.githubusercontent.com/127159714/271940995-d7c534e1-7939-4b10-a2b5-4caa7a803109.png) Buttons not functioning.

Could You Please explain your error?

**Comment by 4artiseth** [comment]
I havent read the code. 
But the issue could be that its unable to scrape data for the products nor is able to do **anything** upon clicking the search button except the "searching..." animation.
The above buttons of "search" "fav" and "user profile" also seem to be just not clickable.

**Comment by NamanMalhotra1706** [comment]
> I havent read the code. But the issue could be that it's unable to scrape data for the products nor is able to do **anything** upon clicking the search button except the "searching..." animation. The above buttons of "search" "fav" and "user profile" also seem to be just not clickable.

Okay, If You Are Not getting Scrape Data, then check your Scraper --> Index.ts, Check Your index.ts file... It will help you to fetch data. if you still facing any issue let me know please

**Comment by gurvirbaraich** [comment]
@4artiseth Have you updated `.env` file with your environment variables as @adrianhajdin will purge all the secret he used in the video.

**Comment by 4artiseth** [comment]
> Okay, If You Are Not getting Scrape Data, then check your Scraper --> Index.ts, Check Your index.ts file... It will help you to fetch data. if you still facing any issue let me know please

HIS website isnt working. 
Check his webiste yourself:
https://pricewise-jsm.vercel.app/

**Comment by NamanMalhotra1706** [comment]
> > Okay, If You Are Not getting Scrape Data, then check your Scraper --> Index.ts, Check Your index.ts file... It will help you to fetch data. if you still facing any issue let me know please
> 
> HIS website isnt working. Check his webiste yourself: https://pricewise-jsm.vercel.app/

Yes, I checked it, It's not showing scraped data but when you inspect it and check the network you can see that, it is fetching data successfully.  Maybe there is a mistake in his deployed site code

**Comment by NamanMalhotra1706** [comment]
> > Okay, If You Are Not getting Scrape Data, then check your Scraper --> Index.ts, Check Your index.ts file... It will help you to fetch data. if you still facing any issue let me know please
> 
> HIS website isnt working. Check his webiste yourself: https://pricewise-jsm.vercel.app/

Yes, I checked it, It's showing scraped data, when you inspect it and check the network you can see that, it is fetching data successfully. and when you scroll down to last you can see the product of that link.

**Comment by 4artiseth** [comment]
> > > Okay, If You Are Not getting Scrape Data, then check your Scraper --> Index.ts, Check Your index.ts file... It will help you to fetch data. if you still facing any issue let me know please
> > 
> > 
> > HIS website isnt working. Check his webiste yourself: https://pricewise-jsm.vercel.app/
> 
> Yes, I checked it, It's showing scraped data, when you inspect it and check the network you can see that, it is fetching data successfully. and when you scroll down to last you can see the product of that link.

oh okay, what could the issue be then?

**Comment by PRIYAM266** [comment]
This is not technically an error because there is no onClick logic applied to these icons. Currently, they are simply images and not functioning as buttons.

**Comment by gurvirbaraich** [comment]
![image](https://github.com/adrianhajdin/pricewise/assets/117713832/a6266a0f-3116-4734-bf0d-45aa206a056e)

The button only shows the loading animation but does not do anything. I believe he has purged all the environment variables that he has used during the video. As he has so many subs all even using once will cost him so much money.

**Comment by Jobex11** [comment]
kindly send the code, I will have a better view on how to fix the function. But I think on Search, is unable to scrape products.

**Comment by Praneshbharadwaj** [comment]
see the product it is geting displayed at the bottom of the page it not redirecting after we searxch the item rather it is just appending down in the trending products sectrion
